scripts/commands/updateApiDocs.ts

```diff
@@ -358,6 +410,11 @@ async function convertHtmlToMarkdown(
     await writeFile(`${markdownPath}/release-notes/index.md`, markdown);
   }
 
+  if (historical) {
+    await $`find ${markdownPath}/release-notes/* -not -path "*index.md" | xargs rm -rf {}`;
+    await $`find docs/api/${pkg.name}/release-notes/* -not -path "*index.md" | xargs -I {} cp -a {} docs/api/${pkg.name}/${versionWithoutPatch}/release-notes/`;
+  }
```

Comment:
Nit: probably move this to a dedicated helper function like `copyReleaseNotes`

Reply:
Done!